src/mon/OSDMonitor.cc

```diff
@@ -7784,22 +7784,57 @@ int OSDMonitor::get_crush_rule(const string &rule_name,
   return 0;
 }
 
-int OSDMonitor::check_pg_num(int64_t pool, int pg_num, int size, ostream *ss)
+int OSDMonitor::check_pg_num(int64_t pool, int pg_num, int size, int crush_rule, ostream *ss)
 {
   auto max_pgs_per_osd = g_conf().get_val<uint64_t>("mon_max_pg_per_osd");
-  auto num_osds = std::max(osdmap.get_num_in_osds(), 3u);   // assume min cluster size 3
-  auto max_pgs = max_pgs_per_osd * num_osds;
   uint64_t projected = 0;
-  if (pool < 0) {
+  unsigned osd_num = 0;
+  auto num_osds = std::max(osdmap.get_num_in_osds(), 3u); 
+  // assume min cluster size 3
```

Comment:
if you want to wrap this line, might want to put the comment above the commented code.